koku/masu/database/sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -94,6 +96,62 @@ CREATE TEMPORARY TABLE matched_tags_{{uuid | sqlsafe}} AS (
 )
 ;
 
+-- Selects the data from aws daily and includes only
+-- the enabled tags in the results
+CREATE TEMPORARY TABLE reporting_aws_with_enabled_tags_{{uuid | sqlsafe}} AS (
+    WITH cte_array_agg_keys AS (
+        SELECT array_agg(key) as key_array
+        FROM {{schema | sqlsafe}}.reporting_awsenabledtagkeys
+    ),
+    cte_filtered_aws_tags AS (
+        SELECT id,
+            jsonb_object_agg(key,value) as aws_tags
+        FROM (
+            SELECT lid.id,
+                lid.tags as aws_tags,
+                aak.key_array
+            FROM {{schema | sqlsafe}}.reporting_awscostentrylineitem_daily lid
+            JOIN cte_array_agg_keys aak
+                ON 1=1
+            WHERE (
+                    lid.tags ?| aak.key_array
+                )
```

Comment:
We'll want to add start date / end date filtering and bill id filtering here so we only grab from the the source we're interested in and the date range we're looking at.

Reply:
added start/end/bill id filtering